fury/actor.py

```diff
@@ -1147,10 +1147,14 @@ def display_extent(self, x1, x2, y1, y2, z1, z2):
                                       peaks_dirs[tuple(center)][i] * pv + xyz))
                     list_dirs.append(symm)
 
-            self.mapper = line(list_dirs, colors=colors,
-                               opacity=opacity, linewidth=linewidth,
-                               lod=lod, lod_points=lod_points,
-                               lod_points_size=lod_points_size).GetMapper()
+            self.line = line(list_dirs, colors=colors,
+                             opacity=opacity, linewidth=linewidth,
+                             lod=lod, lod_points=lod_points,
+                             lod_points_size=lod_points_size)
+
+            self.property = self.line.GetProperty()
```

Comment:
Just for brevity, can we shorten it to: 

```python
self.SetProperty(self.line.GetProperty())
self.SetMapper(self.line.GetMapper())
```

Otherwise, this looks good to me.

Reply:
+1 : cd005b1 applies the requested change.